pkg/devicemapper/devmapper.go

```diff
@@ -264,11 +264,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-// LogInitVerbose is an interface to initialize the verbose logger for the device mapper library.
-func LogInitVerbose(level int) {
```

Comment:
I am not saying somebody is making use of it. Just that it is an interface exported by package and if we remove an existing interface (whether it makes much sense or not), there is always potential to break an existing user out there somewhere.